website/tutorial/testcontainers-with-podman.md

```diff
@@ -0,0 +1,196 @@
+    if (!redisClient) {
+        throw new Error('Redis client is not connected');
+    }
+    return redisClient.get(key);
+}
+
+export async function deleteValue(key: string[]): Promise<number> {
+    if (!redisClient) {
+        throw new Error('Redis client is not connected');
+    }
+    return await redisClient.del(key);
+}
+
+export async function disconnectRedis() {
+    if (redisClient) {
+        await redisClient.quit();
+        redisClient = undefined;
+    }
+}
+```
+
+- Creating basic tests for CRUD operations
+
+```index.spec.ts
+import { afterAll, beforeAll, beforeEach, expect, test } from "vitest";
+import { connectRedis, deleteValue, disconnectRedis, getValue, setValue } from ".";
+import { RedisContainer } from "@testcontainers/redis";
+import { Wait } from "testcontainers";
+import { createClient } from "redis";
+
+beforeAll(() => {
+    new RedisContainer()
+        .withExposedPorts(6379)
+        .withWaitStrategy(Wait.forLogMessage("Ready to accept connections"))
+        .start();
+
+    connectRedis(`redis://localhost:6379`);
+});
+
+afterAll(() => {
+    disconnectRedis();
+});
+
+beforeEach(async () => {
+    // Flushind DB and adding to Redis some values before each test
+    const client = createClient({ url:`redis://localhost:6379` });
+    await client.connect();
+
+    await client.flushDb();
+    await client.set('preset-key', 'preset-value');
+    await client.set('preset-key1', 'preset-value1');
+    await client.quit();
+});
+
+test("set value on server", async () => {
+    // Set value
+    const ret = await setValue("key", "value");
+    expect(ret).toBe("OK");
+
+    // Update value
+    const ret1 = await setValue("key", "updated-value");
+    expect(ret1).toBe("OK");
+});
+
+test("get value from server", async () => {
+    // Get preset value
+    const value = await getValue("preset-key");
+    expect(value).toBe("preset-value");
+
+    // Get not existing value
+    const value1 = await getValue("key");
+    expect(value1).toBeNull();
+});
+
+test("delete value on server", async () => {
+    // Delete two records in a same time
+    const res = await deleteValue(["preset-key", "preset-key1"]);
+    expect(res).toBe(2);
+
+    // Delete not existing record
+    const res1 = await deleteValue(["key"]);
+    expect(res1).toBe(0);
+});
+```
+
+## Running tests
+
+When runnig Testcontainers for a first time I hightly advice to run the tests in DEBUG mode using this command:
+
+```shell-shession
+$ DEBUG=testcontainers* npm test
+```
+
+Then you should be able to see lines such as:
+
+```console
+testcontainers [DEBUG] Loading ".testcontainers.properties" file...
+testcontainers [DEBUG] Loaded ".testcontainers.properties" file
+testcontainers [DEBUG] Found custom configuration: dockerHost: "unix:///run/user/1000//podman/podman.sock
+```
+
+Those lines means that the config file was found by Testcontainers and that containers are using Podman engine instead of Docker.
```

Comment:
Maybe, need a conclusion that points the reader to the project and eventually to other sample code to use?
Also, I think it'd be a great idea to mention to the reader that if they are facing any issue, they should create a ticket.